src/DataLoader/Extensions/SubscribeToData/INotifyCollectionChanged.Extensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Text;
+using Chinook.DataLoader;
+using Chinook.DataLoader.SubscribeToData;
+
+namespace System.Collections.Specialized
+{
+	public static class NotifyCollectionChangedExtensions
+	{
+		public static IDisposable SubscribeToUpdateDataLoader(this INotifyCollectionChanged collection, IDataLoaderSetter setter)
+		{
+			collection.CollectionChanged += OnCollectionChanged;
```

Comment:
-= first?

Reply:
no, the -= is wrapped by an IDisposable. Doing the -= first would result in an unexpected behavior because if you call this method twice, you expect 2 subscriptions.